.gitignore

```diff
@@ -27,3 +27,4 @@ build-stamp
 .pytest_cache/
 .mypy_cache/
 .benchmarks/
+venv
```

Comment:
```suggestion
```
Don't put env specific things here, you can create a global gitignore to not have to add 'venv' or `.idea` in all your projects see https://stackoverflow.com/a/7335487/2519059.